packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -14,19 +14,24 @@ import 'timeline_protocol.dart';
 
 class FramesBarChart extends CoreElement {
   FramesBarChart(TimelineController timelineController)
-      : super('div', classes: 'timeline-frames section-border') {
+      : super('div', classes: 'timeline-frames') {
     layoutHorizontal();
     element.style
       ..alignItems = 'flex-end'
       ..height = '${chartHeight}px'
       ..paddingTop = '${topPadding}px';
 
-    timelineController.onFrameAdded.listen((TimelineFrame frame) {
-      if (frameUIgraph == null) {
-        frameUIgraph = PlotlyDivGraph(this, frame, false); // Process chunks
-        add(frameUIgraph);
-      }
+    frameUIgraph = PlotlyDivGraph(this, timelineController);
+    add(frameUIgraph);
 
+    // Make sure DIV exist.
+    // TODO(terry): Use requestAnimationFrame instead?
```

Comment:
I would copy the setState method you will see elsewhere in the code.
The body of the setState call will be this code you are running slightly async.
That will make converting to flutter web slightly easier.

Reply:
Done, thanks for the info.